### ISSUE TITLE
deprecation warning in passlib

### ISSUE THREAD

**Issue by tristanmkernan** [dusktreader/flask-praetorian#151]
in running tests i get a lot of repeated warnings from flask-praetorian:

```
DeprecationWarning: the method passlib.context.CryptContext.encrypt() is deprecated as of Passlib 1.7, and will be removed in Passlib 2.0, use CryptContext.hash() instead.
    return self.pwd_ctx.encrypt(raw_password, scheme=self.hash_scheme)
```

just a heads up and thanks for the great lib!

**Comment by dusktreader** [comment]
This will be addressed in the new release. Working out some of the kinks now, so look for 1.0 soonish!

**Comment by ClintOxx** [comment]
@dusktreader do you have a general idea on when 1.0 will come out and if it will have breaking changes?

**Comment by dusktreader** [comment]
I am hoping to have 1.0 done this month. There should not be breaking changes.

**Comment by ClintOxx** [comment]
Sounds good thanks

**Comment by dusktreader** [comment]
Resolved by pr #167
